src/components/crewretrieval.tsx

```diff
@@ -0,0 +1,296 @@
+			.then(allCrew => this.setState({ allCrew }));
+	}
+
+	componentDidUpdate() {
+		if (!this.state.ownedPolestars || !this.state.allCrew || this.state.data) {
+			return;
+		}
+		let data = this.state.allCrew.filter(
+			(crew) => crew.unique_polestar_combos?.some(
+				(upc) => upc.every(
+					(trait) => this.state.ownedPolestars.some(op => filterTraits(op, trait))
+				)
+			)
+		);
+		this.setState({ data });
+	}
+
+	_onChangePage(activePage) {
+		this.setState({ pagination_page: activePage });
+	}
+
+	_compare(a, b) {
+		return (a > b ? 1 : b > a ? -1 : 0);
+	}
+
+	_handleSort(clickedColumn, isSkill) {
+		const { column, direction } = this.state;
+		let { data } = this.state;
+
+		if (column !== clickedColumn) {
+			let sortedData;
+			if (isSkill) {
+				sortedData = data.sort(
+					(a, b) =>
+						(a.base_skills[clickedColumn] ? a.base_skills[clickedColumn].core : 0) -
+						(b.base_skills[clickedColumn] ? b.base_skills[clickedColumn].core : 0)
+				);
+			} else {
+				sortedData = data.sort((a, b) => this._compare(a[clickedColumn], b[clickedColumn]));
+			}
+
+			this.setState({
+				column: clickedColumn,
+				direction: 'ascending',
+				pagination_page: 1,
+				data: sortedData
+			});
+		} else {
+			this.setState({
+				direction: direction === 'ascending' ? 'descending' : 'ascending',
+				pagination_page: 1,
+				data: data.reverse()
+			});
+		}
+	}
+
+	_findCombosForCrew(crew: any) {
+		let combos = crew.unique_polestar_combos?.filter(
+			(upc) => upc.every(
+				(trait) => this.state.ownedPolestars.some(op => filterTraits(op, trait))
+			)
+		).map((upc) => upc.map((trait) => this.state.ownedPolestars.find((op) => filterTraits(op, trait))));
+		return (
+			<div className="ui accordion" onClick={() => this.setState({ activeCrew: this.state.activeCrew === crew.symbol ? null : crew.symbol })}>
+				<div className={`title ${this.state.activeCrew === crew.symbol ? 'active' : ''}`}>View</div>
+				<div className={`content ${this.state.activeCrew === crew.symbol ? 'active' : ''}`}>
+						<div className="ui four column grid">
+							{combos.map((combo) => (
+								<div className="row">
+									{combo.map((polestar) => (
+											<div className="column"><img width={32} src={`${process.env.GATSBY_ASSETS_URL}${polestar.icon.file.substr(1).replace(/\//g, '_')}`} /><br />{polestar.name.replace(' Polestar', '').replace(' Skill', '')}</div>
+									))
+									}
+								</div>
+							))}
+					</div>
+				</div>
+			</div>
+		)
+	}
+
+	render() {
+		const { column, direction, pagination_rows, pagination_page, unownedOnly, minRarity } = this.state;
+		let { data } = this.state;
+		if (!data) {
+			return null;
+		}
+
+		let totalPages = Math.ceil(data.length / this.state.pagination_rows);
+		
+		if (unownedOnly) {
+			data = data.filter((crew) => !this.props.playerData.player.character.crew.some((c) => crew.symbol === c.symbol));
+		}
+
+		if (minRarity) {
+			data = data.filter((crew) => crew.max_rarity >= minRarity);
+		}
```

Comment:
Small bug: totalPages should be calculated after filtering by unowned and rarity

```suggestion
		if (unownedOnly) {
			data = data.filter((crew) => !this.props.playerData.player.character.crew.some((c) => crew.symbol === c.symbol));
		}

		if (minRarity) {
			data = data.filter((crew) => crew.max_rarity >= minRarity);
		}
		
		let totalPages = Math.ceil(data.length / this.state.pagination_rows);
```